.editorconfig

```diff
@@ -52,7 +52,7 @@ indent_size = 2
 
 # Feature files
 [*.feature]
-end_of_line = lf
+end_of_line = crlf
 insert_final_newline = true
```

Comment:
Are we going back to just good old crlf ?

Reply:
Thought try something new because we are still getting inconsistent line endings 😢

Reply:
ok